src/views/consent/ConsentBeacon.js

```diff
@@ -31,19 +31,25 @@ define(['okta'], function (Okta) {
       </div>\
       <div class="logo-wrapper consent-beacon-client">\
         <div class="consent-beacon-border"/>\
```

Comment:
looked at line 20, looks like client img/uri shall be nested in `consent-beacon-border`.
otherwise why need an empty container?

Reply:
Was trying to follow the pattern of the other Beacons we have. https://github.com/okta/okta-signin-widget/blob/master/src/views/shared/SecurityBeacon.js#L153 border is inside 'auth-beacon-security' where we append the image.